easybuild/easyconfigs/d/DBD-mysql/DBD-mysql-4.050-foss-2020b-Perl-5.32.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'PerlModule'
+
+name = 'DBD-mysql'
+version = '4.050'
+versionsuffix = '-Perl-%(perlver)s'
```

Comment:
Please drop the versionsuffix (we dropped this for perl in newer toolchains as it was superfluous).

Reply:
suffix dropped,